packages/gatsby/src/utils/webpack.config.js

```diff
@@ -316,12 +316,27 @@ module.exports = async (
     switch (stage) {
       case `develop`:
         return `cheap-module-source-map`
-      // use a normal `source-map` for the html phases since
+      // use a normal `source-map` for the develop-html phase since
       // it gives better line and column numbers
       case `develop-html`:
-      case `build-html`:
-      case `build-javascript`:
         return `source-map`
+
+      // Source maps expose too much information about the site so
+      // we don't generate them by default.
+      //
+      // You can easily enable them in your `gatsby-node.js` but make
+      // sure that your web server restricts access to them (e.g. with
+      // basic auth):
+      //
+      // exports.modifyWebpackConfig = ({ config, stage }) => {
```

Comment:
The API for modifying webpack config has changed. Instead of directly mutating the webpack config, there's a new action named `setWebpackConfig`. If you grep the v2 branch, you'll find a number of usages for this + I think there's docs already.

Reply:
Ok, fixed.